app/blogHandler.go

```diff
@@ -0,0 +1,134 @@
+package app
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/maayarosama/Blogging_system/internal"
+	"github.com/maayarosama/Blogging_system/models"
+	"github.com/rs/zerolog/log"
+)
+
+func (a *App) GetBlogs(w http.ResponseWriter, r *http.Request) {
+	blogs := a.db.GetBlogs()
+	res, _ := json.Marshal(blogs)
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(res)
+}
+func (a *App) GetUsersBlogs(w http.ResponseWriter, r *http.Request) {
+	userID := r.Context().Value(internal.UserIDKey("UserID")).(int)
+	blogs := a.db.GetUsersBlogs(userID)
+	res, _ := json.Marshal(blogs)
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(res)
```

Comment:
setting headers with status and a body is repeated in all handlers, a helper would be better.